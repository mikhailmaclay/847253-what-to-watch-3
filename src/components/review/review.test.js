// Libraries
import React from 'react';
import renderer from 'react-test-renderer';
// Components
import Review from './review';

const review = {
  user: 5,
  rating: 5,
  date: `November 18, 2020 03:24:00`,
  text: `DO! IT!`
};

it(`<Review/> should render correctly`, () => {
  const {user, rating, date, text} = review;

  const result = renderer.create(
      <Review rating={rating} user={user} date={date} text={text}/>
  );

  expect(result).toMatchSnapshot();
});
