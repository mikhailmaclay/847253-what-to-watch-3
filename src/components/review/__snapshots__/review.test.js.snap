// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Review/> should render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      DO! IT!
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Amanda Greever
      </cite>
      <time
        className="review__date"
        dateTime="November 18, 2020 03:24:00"
      >
        November 18, 2020
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    5.0
  </div>
</div>
`;
